Zilon.Core/Zilon.Core.Specs/Steps/CommonSteps.cs

```diff
@@ -228,13 +227,29 @@ public CommonSteps(CommonGameActionsContext context) : base(context)
             propTransferCommand.Execute();
         }
 
+        [When(@"Жду (.*) единиц времени")]
+        public async System.Threading.Tasks.Task WhenЖдуЕдиницВремениAsync(int timeUnitCount)
+        {
+            await WhenСледующаяИтерацияСектораAsync(timeUnitCount);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)